lib/src/prices/get_prices_parameters.dart

```diff
@@ -53,6 +52,11 @@ class GetPricesParameters extends GetParametersHelper<GetPricesOrderField> {
     addNonNullDate(createdGte, 'created__gte', dayOnly: false);
     addNonNullDate(createdLte, 'created__lte', dayOnly: false);
     addNonNullString(kind?.offTag, 'kind');
+    if (additionalParameters != null) {
+      for (final String key in additionalParameters!.keys) {
+        addNonNullString(additionalParameters![key], key);
+      }
+    }
```

Comment:
Please:
1. move that to the super method
2. use `_result.addAll(additionalParameters);` instead